drivers/SmartThings/zigbee-switch/src/thirdreality/init.lua

```diff
@@ -0,0 +1,31 @@
+local capabilities = require "st.capabilities"
+
+local THIRDREALITY_MOTIOM_CLUSTER = 0xFC00
```

Comment:
Fix spelling here and below
```suggestion
local THIRDREALITY_MOTION_CLUSTER = 0xFC00
```